testcases/MachineConfig.py

```diff
@@ -79,6 +80,17 @@ def runTest(self):
                 status = CecConfig(self.cv_HMC,self.system_name,self.lpar_name,self.lpar_prof,lmb_size).CecSetup_lmb()
                 if status:
                     self.fail(status)
+            if "eio" in config_value:
+                if self.bmc_type == "FSP_PHYP":
+                    eio_slot_count = re.findall('eio=[0-9]+', str(self.machine_config))[0].split('=')[1]
+                    if int(eio_slot_count) not in range(9):
+                        self.skipTest("%s is not valid eio slot count, "
+                                      "Please pass a number between 1 to 8, and pass 0 to disable Enlarged IO" % eio_slot_count)
+                    self.cv_HMC.poweroff_system()
+                    self.cv_FSP.cv_ASM.configure_enlarged_io(eio_slot_count)
+                    self.cv_HMC.poweron_system()
```

Comment:
power off once for all cec changes and than poweron at once after chaning lmb, hugepage and io enlarge